runtime-light/crypto/crypto-builtins.cpp

```diff
@@ -0,0 +1,28 @@
+//  Compiler for PHP (aka KPHP)
+//  Copyright (c) 2024 LLC «V Kontakte»
+//  Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/crypto/crypto-builtins.h"
+
+#include "runtime-light/tl/tl-core.h"
+#include "runtime-light/streams/interface.h"
+
+// Crypto-Specific TL magics
+constexpr uint32_t TL_GET_CRYPTOSECURE_PSEUDORANDOM_BYTES = 0x2491'b81d;
```

Comment:
Please, put in anonymous namespace. It's not gonna be used by other parts of runtime and we don't want this symbol to be exported.